core/src/main/java/io/grpc/internal/ClientStream.java

```diff
@@ -46,7 +46,7 @@
    * period until {@link ClientStreamListener#closed} is called. This method is safe to be called
    * at any time and multiple times.
    *
-   * @param reason must be one of Status.CANCELLED or Status.DEADLINE_EXCEEDED
+   * @param reason must have Code CANCELLED, DEADLINE_EXCEEDED, INTERNAL, or UNKNOWN
```

Comment:
I tried it out in eclipse and got javadoc's invalid qualifier warning. Some quick searching on SO yesterday seemed to indicate that it was not possible, so I left it.  Fully qualifying the name works.